src/main/java/com/bankbazaar/test/service/DepartmentService.java

```diff
@@ -50,8 +43,14 @@ public Department getDepartmentByName(String name) {
     }
     /**
      * Delete record by id
+     * @param id
      */
-    public String deleteDepartment(int id) {
+    public String deleteDepartment(long id) {
+        Department existingDepartment = departmentRepository.findById(id).orElse(null);
+        if(existingDepartment==null)
```

Comment:
if (departmentRepository.findById(id).isEmpty())

Reply:
i haven't made changes to this part as "existingDepartment" object is used for updating Department